api/hasura/actions/_handlers/updateCircleStartingGive.ts

```diff
@@ -0,0 +1,101 @@
+import assert from 'assert';
+
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import { z } from 'zod';
+
+import { authCircleAdminMiddleware } from '../../../../api-lib/circleAdmin';
+import { adminClient } from '../../../../api-lib/gql/adminClient';
+import { getInput } from '../../../../api-lib/handlerHelpers';
+import {
+  errorResponseWithStatusCode,
+  UnprocessableError,
+} from '../../../../api-lib/HttpError';
+
+const updateCircleStartingGiveInput = z
+  .object({
+    circle_id: z.number(),
+    starting_tokens: z.number().optional(),
+  })
+  .strict();
+
+async function handler(req: VercelRequest, res: VercelResponse) {
+  const { payload } = await getInput(req, updateCircleStartingGiveInput, {
```

Comment:
we could define `circle_id` right here via nested destructuring and not need line 27

Reply:
I am using payload later. 
I will destructure it and use the elements in the rest of the code